portal/time_tracker/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+class Time_tracker(models.Model):
+
+    EVENT = "event"
+    CLASS = "class"
+    OTHER = "other"
+
+    TIME_TYPE_CHOICES = (
+        (EVENT, 'Event'),
+        (CLASS, 'Class'),
+        (OTHER, 'Other')
+    )
+
+    hours = models.IntegerField()
+    notes = models.charField(max_length=1000)
```

Comment:
Because this can be many lines of text, this could be a `TextField`, with no `max_length`.

https://docs.djangoproject.com/en/1.11/ref/models/fields/#textfield